src/execution/service.ts

```diff
@@ -818,6 +827,11 @@ export class ServiceScriptExecution extends BaseExecutionWithCommand<ServiceScri
   private _onChildStarted() {
     switch (this._state.id) {
       case 'starting': {
+        this._logger.log({
+          script: this._config,
+          type: 'info',
+          detail: 'service-process-started',
```

Comment:
Nit: Is it worth reflecting the state names in `detail`? E.g., `service-process-starting`?

Reply:
Starting makes sense to the internal state machine of ServiceScriptExecution, but for everywhere else at this point we have a child process now, so the process has started.